tests/DynamoDbClientServiceTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * Class DynamoDbClientServiceTest
+ */
+class DynamoDbClientServiceTest extends TestCase
```

Comment:
Well I can add a PSR-4 namespace to the whole test dir but I didn't really think that was in the scope of what was trying to be achieved here.